tsconfig.json

```diff
@@ -1,16 +1,14 @@
 {
 	"compilerOptions": {
 		"outDir": "${configDir}/distribution",
-		"module": "node16",
-		"moduleResolution": "node16",
+		"module": "preserve",
 		"moduleDetection": "force",
 		"target": "ES2022", // Node.js 18
 		"lib": [
 			"DOM",
 			"DOM.Iterable",
 			"ES2022"
 		],
-		"resolveJsonModule": false, // ESM doesn't yet support JSON modules.
```

Comment:
Not a necessary change, but `preserve` implies this, if set.

I think this can be dropped even as a non-breaking change, as long as you _don't_ consider tsconfig to **enforce** a minimum Node version. i.e. if the user uses import assertions, let them. It might be worth updating the "Node.js 18" comment above though